test/xpu_api/numerics/c.math/cmath.pass.cpp

```diff
@@ -120,21 +117,17 @@ void test_ceil()
         static_assert((std::is_same<decltype(dpl::ceil((unsigned long)0)), double>::value), "");
         static_assert((std::is_same<decltype(dpl::ceil((long long)0)), double>::value), "");
         static_assert((std::is_same<decltype(dpl::ceil((unsigned long long)0)), double>::value), "");
-        static_assert((std::is_same<decltype(dpl::ceil((double)0)), double>::value), "");
-    };
-    IF_DOUBLE_SUPPORT_L(fnc)
```

Comment:
Do lambdas have any value for testing? Can we get rid of it?

Reply:
No, tis lambda absent in initial test variant.
It was only implementation detail of current variant of test.